x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/EqlClientException.java

```diff
@@ -8,16 +8,12 @@
 
 import org.elasticsearch.xpack.ql.QlClientException;
 
-public abstract class EqlClientException extends QlClientException {
+public class EqlClientException extends QlClientException {
```

Comment:
thought: this will make the exception show in the REST response as `"type": "eql_client_exception"`. Ideally, we probably want to prefer slightly more specific subclasses of this (`VerificationException` is fine, for instance), but IMO this is a good solution for now.

Reply:
The `VerificationException` is quite specific, though, requiring a `Failure`s collection, each requiring a `Source` which isn't available everywhere.